components/automate-chef-io/content/docs/iam-v2-overview.md

```diff
@@ -152,23 +151,102 @@ The default policies are: Viewers, Editors, Administrator, and Ingest.
 
 Just add members to any of these default policies, as described later, to quickly get your basic users up and running.
 
-### Policies and Roles in Automate
+## Working with Projects
 
-<!-- I do not like saying "user interface" because technically that includes the command line, too. But I can live with it if you think that is the lesser of two evils. -->
-IAM v2 introduces partial policy management from within Chef Automate in your browser.
+IAM projects are collections of resources created in Chef Automate or ingested from external data providers, including ChefInfra and ChefInspec.
```

Comment:
have we made the move to referring to chef infra in the automate docs? also i think you need a space in those names